Doc/library/queue.rst

```diff
@@ -151,6 +151,14 @@ provide the public methods described below.
    else raise the :exc:`Empty` exception (*timeout* is ignored in that case).
 
 
+   Prior to 3.0 on POSIX systems, and for all versions on Windows, if
+   *block* is true and *timeout* is ``None``, this operation goes into
+   an uninterruptible wait on an underlying lock.
+
```

Comment:
Nit: Join these two sentences into a single paragraph.  The second sentence depends on the first and is the crux of the point being made.